R/random-forest-development.R

```diff
@@ -93,12 +94,14 @@ source('R/supervised-model-development.R')
 #' lasso <- LassoDevelopment$new(p)
 #' lasso$run()
 #'
+#' set.seed(42) 
 #' # Run Random Forest
 #' rf <- RandomForestDevelopment$new(p)
 #' rf$run()
 #'
 #' print(proc.time() - ptm)
 #'
+#' \donttest{
 #' #### Example using SQL Server data #### This example requires: 1) That you alter
```

Comment:
added this comment to each don't test example:
#' #### This example is specific to Windows and is not tested.